### DISCUSSION TITLE
Application Publishing

### DISCUSSION THREAD

**Discussion by thomasp7983** [Hexer10/youtube_explode_dart#190] [Q&A] [answered]
Does Apple or Google reject the application if using this Library?

**Comment by Hexer10** [comment, marked as answer]
Read here https://github.com/Hexer10/youtube_explode_dart/issues/152

**Comment by adambak033** [comment]
Hey there, sorry if it is not the right place to ask but, do you have any update regarding this request ?
Do stores accept the app if you use the download and save into files features ?
